src/main/groovy/org/shipkit/internal/gradle/TravisPlugin.java

```diff
@@ -40,7 +41,9 @@ public void apply(final Project project) {
             @Override
             public void execute(GitBranchPlugin p) {
                 IdentifyGitBranchTask identifyBranch = (IdentifyGitBranchTask) project.getTasks().getByName(IDENTIFY_GIT_BRANCH);
-                identifyBranch.setBranch(branch);
+                if(StringUtils.isNotEmpty(branch)) {
```

Comment:
Can you use our own StringUtil instead? This way we avoid dependency on apache commons (less dependencies -> less problems for our users).